src/sentry/replays/usecases/ingest/__init__.py

```diff
@@ -85,7 +76,7 @@ def ingest_recording(message_dict: RecordingMessage, transaction: Span, current_
                 project_id=message_dict["project_id"],
                 received=message_dict["received"],
                 retention_days=message_dict["retention_days"],
-                payload_with_headers=message_dict["payload"],
+                payload_with_headers=cast(bytes, message_dict["payload"]),
```

Comment:
@untitaker I assume there's a limitation that prevents this from being typed at the schema level.  Is this a work in progress or not possible?

Reply:
bytestrings are `Any`-typed in the schema. there are probably ways to fix that (such as defining proprietary extensions to jsonschema) but we decided it was not worth it